rllib/evaluation/worker_set.py

```diff
@@ -397,6 +398,10 @@ def sync_weights(
                 weights to. If None (default), sync to all remote workers.
             global_vars: An optional global vars dict to set this
                 worker to. If None, do not update the global_vars.
+            timeout_seconds: Timeout in seconds to wait for the sync weights
```

Comment:
What would None do then?

Reply:
indefinitely until all the object_refs are ready.
this is actually all standard ray.wait behavior. look at documentation for ``ray.wait(timeout=...)``.
https://docs.ray.io/en/latest/ray-core/package-ref.html#ray-wait
the default timeout is None.